<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSVDialog</class>
 <widget class="QWidget" name="CSVDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton_export">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Exporter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_quitte">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remarque: le dernier objet enregistré est exporté.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>421</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_showDebitPatient">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Débit Patient</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_showTopDeJour">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Top de Jour</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_showScan3D">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Scan 3D</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_showProfile">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Profile</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_showPicDeBragg">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Pic de Bragg</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_showSOBP">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>SOBP</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
